std/assembly/typedarray.ts

```diff
@@ -1027,6 +1159,53 @@ function FIND_INDEX<TArray extends ArrayBufferView, T>(
   return -1;
 }
 
+// @ts-ignore: decorator
+@inline
+function INCLUDES<TArray extends ArrayBufferView, T>(
+  array: TArray,
+  searchElement: T,
+  fromIndex: i32,
+): bool {
+  return INDEX_OF<TArray, T>(array, searchElement, fromIndex) >= 0;
```

Comment:
Just a question about the implementation here. Is there a reason why it's not this:

```ts
return INDEX_OF<TArray, T>(array, searchElement, fromIndex) ^ 0
```

Reply:
`x ^ 0 == x` so this unnecessary. May be you messed up with `x ^ -1` or `~x` (which the same)? But this trick make sense only for specific cases, bit not in general

Reply:
Oh yeah! It should return `-1` when it can't find the index.